File: src/main/java/io/bloco/faker/components/Food.java
package io.bloco.faker.components;

import io.bloco.faker.FakerComponent;
import io.bloco.faker.FakerData;

public class Food extends FakerComponent {

    public Food(FakerData data) {
        super(data);
    }

    public String dish() {
        return fetch("food.dish");
    }

    public String description() {
        return fetch("food.descriptions");
    }

    public String ingredient() {
        return fetch("food.ingredients");
    }

    public String spice() {
        return fetch("food.spices");
    }

    public String measurement() {
        return fetch("food.measurement_sizes") + " " + fetch("food.measurements");
    }

    public String metricMeasurement() {
        return fetch("food.metric_measurements");
    }
}

File: lib/src/main/kotlin/io/bloco/faker/components/Food.kt
package io.bloco.faker.components

import io.bloco.faker.FakerComponent
import io.bloco.faker.FakerData

class Food(data: FakerData) : FakerComponent(data) {
    fun dish(): String {
        return fetch("food.dish")
    }

    fun description(): String {
        return fetch("food.descriptions")
    }

    fun ingredient(): String {
        return fetch("food.ingredients")
    }

    fun spice(): String {
        return fetch("food.spices")
    }

    fun measurement(): String {
        return fetch("food.measurement_sizes") + " " + fetch("food.measurements")
    }

    fun metricMeasurement(): String {
        return fetch("food.metric_measurements")
    }
}